GRADLE_STRUCTURE.md

```diff
@@ -0,0 +1,16 @@
+# Gradle Structure
+
+package/example/android/settings.gradle imports the flutter tooling, includes the app directory, and configures GoogleCloudPlatform/artifact-registry-maven-tools for use in ci.
```

Comment:
Actually I just realized that the ` marks around package/example/android/settings.gradle were lost in accepting the changes!